app/src/main/java/cm/aptoide/pt/ApplicationModule.java

```diff
@@ -1095,8 +1095,8 @@ AnalyticsBodyInterceptorV7 provideAnalyticsBodyInterceptorV7(
         .build();
   }
 
-  @Singleton @Provides @Named("retrofit-wallet") Retrofit providesV8SecondaryRetrofit(
-      @Named("v8") OkHttpClient httpClient, @Named("base-secondary-host") String baseHost,
+  @Singleton @Provides @Named("retrofit-v7-secondary") Retrofit providesV8SecondaryRetrofit(
```

Comment:
dagger annotation name is v7 and method is v8. suggestion: provdesV8SecondaryRetrofit